core-services/release-controller/_releases/release-ocp-4.15.json

```diff
@@ -32,22 +32,22 @@
     }
   },
   "verify": {
-    "agent-compact-ipv4": {
+    "agent-compact-ipv4-conformance": {
```

Comment:
as the for job label (since it will get displayed in sippy), I'd keep it without `-conformance` (the `prowJob` one will keep the `-conformance` for the match)